index.d.ts

```diff
@@ -218,7 +218,7 @@ export type DateTimePickerProps = {
   labelProps?: LabelProps;
   datePickerProps?: { [key: string]: any };
   timePickerProps?: { [key: string]: any };
-  onTimeInputBlur?: () => void;
+  onTimeInputBlur?: (dateTime: Dayjs) => void;
```

Comment:
Can we rename it to `onBlur`? I understand that we are passing this prop to `TimePickerInput`. But for a user using this component, `DateTimePicker` would be like a single component. So, in that case, if we lose focus from the component, we need to trigger this function. It is working as expected, we just need to rename the prop. WDYT?

Reply:
I thought of the same. But when I was testing in a product(neeto-planner), I saw they already using the `onTimeInputBlur`, So this will become a breaking change, If this can be handled while rolllout I can rename to just `onBlur`.

Reply:
In that case, add the `onBlur` prop without removing the `onTimeInputBlur` prop. When you roll out, replace `onTimeInputBlur` with `onBlur`. Once the rollout is complete, we can remove the `onTimeInputBlur` prop.